### PR TITLE
Update Babel to IE8 (fixes #239)

### LINKED ISSUE #239: IE8 compatibility
The `es3-member-expression-literals` and`es3-property-literals` babel plugins need to be installed to support IE8
https://babeljs.io/docs/plugins/#transform-plugins-es3

### PR DESCRIPTION
**Pull request by incompl**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["es2015", {"loose": true}]],
-  "plugins": ["transform-object-assign"]
+  "presets": ["es3", ["es2015", {"loose": true}]],
+  "plugins": []
```
Comment (gkatsev) [inline .babelrc]: is `transform-object-assign` no longer used? If so, can probably just remove this property altogether.

Comment (incompl) [inline .babelrc]: My understanding is that es3 obviates transform-object-assign

Comment (gkatsev) [inline .babelrc]: I don't think the es3 preset includes it. If `Object.assign` is used anywhere, we still need it.

**Comment by ldayananda** [comment]
Replaced by #244

**Pull request closed without merging**